lib/command/terraform/generate.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Command
+  module Terraform
+    class Generate < ::Command::Base
+      SUBCOMMAND = "terraform"
+      NAME = "generate"
+      DESCRIPTION = "Generates terraform configuration files"
+      LONG_DESCRIPTION = <<~DESC
+        Generates terraform configuration files based on `controlplane.yml` and `templates/` config
```

Comment:
For `LONG_DESCRIPTION` we generally use bullet points.

```suggestion
        - Generates terraform configuration files based on `controlplane.yml` and `templates/` config
```